ci/webhook-receiver.py

```diff
@@ -195,13 +216,28 @@ def init_ci_pipeline():
     out, err = proc2.communicate()
```

Comment:
Technically if CI has finished a run and is in the middle of spawning a call this out value will be wrong anyway, why don't we use that + length of ci request queue to properly determine if we can run instantly